providers/class-two-factor-fido-u2f-admin-list-table.php

```diff
@@ -96,7 +102,7 @@ protected function display_tablenav( $which ) {
 	 */
 	public function single_row( $item ) {
 		?>
-		<tr id="key-<?php echo esc_attr( $item->keyHandle ); ?>">
+		<tr id="key-<?php echo esc_attr( $item->key_handle ); ?>">
```

Comment:
This would need to be changed back if we're keeping WPCS out of the vendor libraries.